modules/openapi-generator/src/main/resources/kotlin-spring/libraries/spring-boot/buildGradle-sb3-Kts.mustache

```diff
@@ -58,6 +58,6 @@ dependencies {
         exclude(module = "junit")
     }
 {{#reactive}}
-    testImplementation`("org.jetbrains.kotlinx:kotlinx-coroutines-test:$kotlinxCoroutinesVersion")
+    testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:$kotlinxCoroutinesVersion")
```

Comment:
This is the primary change.

All other changes were produced by running the requested commands in Git Bash for Windows.